docs/ref/databases.txt

```diff
@@ -1148,7 +1174,7 @@ file::
     DATABASES = {
         'default': {
             'ENGINE': 'mydbengine',
-            ...
+            ...: ...,
```

Comment:
In these cases, is using a comment available: 

```
         # ...
```

Update: I see https://github.com/django/django/pull/16496/commits/a0b3803930195dbe062427ebf662b63e3956197d#r1084661525 makes the same point.

Reply:
Yep changed that, thanks.

The issue is that after changing this I still get these errors on `make html`:

```
docs/ref/logging.txt:20: ERROR: Unexpected indentation.
docs/ref/logging.txt:290: ERROR: Unexpected indentation.
docs/ref/logging.txt:26: WARNING: Definition list ends without a blank line; unexpected unindent.
docs/ref/logging.txt:27: WARNING: Block quote ends without a blank line; unexpected unindent.
docs/ref/logging.txt:29: WARNING: Block quote ends without a blank line; unexpected unindent.
docs/ref/logging.txt:39: ERROR: Unexpected indentation.
docs/ref/logging.txt:309: ERROR: Unexpected indentation.
docs/ref/logging.txt:45: WARNING: Definition list ends without a blank line; unexpected unindent.
docs/ref/logging.txt:46: WARNING: Block quote ends without a blank line; unexpected unindent.
docs/ref/logging.txt:48: WARNING: Block quote ends without a blank line; unexpected unindent.
docs/ref/settings.txt:1696: ERROR: Unexpected indentation.
```

cc @pauloxnet

Reply:
OK, we can look at that 👍 


(Let's see if we can get the code-block adjustments merged.)

Reply:
I "fixed" the `ref/settings.txt` issue.

But for the `logging.txt` ones, I can't spot the solution. But I'm no expert, so it might be an obvious mistake I'm making.

Reply:
Btw, I'm trying to get the "first" part, i.e. have `blacken-docs` work without errors for any of these files.

We can see if it makes sense to have those changes merged. I.e. without the `blacken-docs` changes.

That commit would include more than the `code-block` changes. But not much more.

If we need a "clean" `code-block` changes only commit, I'll create a new PR with that commit only. But first I need to fix the errors (possibly only one error causing in the rest) in `ref/logging.txt` 👀

Reply:
@jvzammit — get it as close as you can, with the energy/time/enthusiasm you have (leaving a buffer! 😅) and I can have a look at any lingerers.

Reply:
I tried it locally and I confirm the same errors/warnings related to `docs/ref/logging.txt` but I can't spot the solution either.

Reply:
Thanks @pauloxnet the indentation changes you suggested fixed the errors 🎉 

@carltongibson @pauloxnet I pushed the changes and "a second commit" with the automatic changes by running:

```
blacken-docs --rst-literal-blocks ref/clickjacking.txt
blacken-docs --rst-literal-blocks ref/databases.txt
blacken-docs --rst-literal-blocks ref/django-admin.txt
blacken-docs --rst-literal-blocks ref/logging.txt
blacken-docs --rst-literal-blocks ref/request-response.txt
blacken-docs --rst-literal-blocks ref/settings.txt
blacken-docs --rst-literal-blocks ref/signals.txt
blacken-docs --rst-literal-blocks ref/template-response.txt
blacken-docs --rst-literal-blocks ref/unicode.txt
blacken-docs --rst-literal-blocks ref/urlresolvers.txt
blacken-docs --rst-literal-blocks ref/utils.txt
```

I can drop that second commit if it expedites merging the first commit 🚀 
After all we can just run the command later on (in a subsequent PR).

Reply:
I think the second commit is ok. I only add one suggestion.

Reply:
Thanks @pauloxnet I dropped the second commit, apply your correct suggestion, fixup and force push. That puts all "manual" changes prior to running the command in the first commit.

Then ran the command again, and put the changes by the command in the second commit.

How is it looking now?